serveradmin/servershell/static/js/servershell/command.js

```diff
@@ -235,6 +235,31 @@ servershell.commands = {
         // Now trigger reload with all changes at once
         servershell.shown_attributes = shown_attributes;
     },
+    bookmarkadd: function(name) {
```

Comment:
Just a suggestion. Wouldn't it make more sense if `bookmark` would add/set a bookmark and `bookmarkload` or `load_bookmark` or `loadbookmark` would load it?
Just about wording, when I talk about "bookmark" then I think about setting a bookmark and not checking one.

Reply:
Sure why not I don't have a strong preference - if it is more intuitive for you :shrug: ... let me search and replace.

Reply:
I'm not a heavy serveradmin user, so whatever works better for the user. Was just my thinking for it already. ^^

Reply:
Lets not overthink it. It is already adjusted.